client/components/editor/Sidebar/index.vue

```diff
@@ -64,9 +67,14 @@ export default {
     selectedActivity: { type: Object, required: true },
     selectedElement: { type: Object, default: null }
   },
-  data: () => ({ selectedTab: 'browser', commentCount: 0 }),
+  data: () => ({ selectedTab: 'browser', commentCount: 0, unseenCount: 0 }),
   computed: {
     selectedTabIndex: vm => vm.tabs.map(it => it.name).indexOf(vm.selectedTab),
+    ...mapState({ seenByActivity: state => state.seenByActivity }),
+    ...mapGetters('repository/comments', ['getActivityComments', 'getUnseenComments']),
```

Comment:
'getActivityComments' => You get this on change event above. Update 
`@change="comments => commentCount = comments.length"` line.